src/server/stream_family.cc

```diff
@@ -432,6 +432,7 @@ int StreamAppendItem(stream* s, CmdArgList fields, uint64_t now_ms, streamID* ad
     /* Get a reference to the tail node listpack. */
     lp = (uint8_t*)ri.data;
     lp_bytes = lpBytes(lp);
+    CHECK_GT(lp_bytes, 0U);
```

Comment:
We already hit it with the core dump and we know it was zero. It's not a harm to keep this. I will follow up adding more of these around trim, del